packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/resolution/ResolutionFeature.java

```diff
@@ -144,6 +143,8 @@ static Size computeBestPreviewSize(int cameraId, ResolutionPreset preset)
    * @return The best possible {@link android.media.CamcorderProfile} that matches the supplied
    *     {@link ResolutionPreset}.
    */
+  @TargetApi(Build.VERSION_CODES.R)
+  @SuppressWarnings({"fallthrough", "deprecation"})
```

Comment:
Apparently the order is important, `@SuppressWarnings({"deprecation", "fallthrough"})` didn't work for some reason.